src/plugins/notebook/components/NotebookEntry.vue

```diff
@@ -144,8 +146,17 @@ import TextHighlight from '../../../utils/textHighlight/TextHighlight.vue';
 import { createNewEmbed } from '../utils/notebook-entries';
 import { saveNotebookImageDomainObject, updateNamespaceOfDomainObject } from '../utils/notebook-image';
 
+import sanitizeHtml from 'sanitize-html';
+import { v4 as uuid } from 'uuid';
 import Moment from 'moment';
 
+const SANITIZATION_SCHEMA = {
+    allowedTags: ['a'],
+    allowedAttributes: {
+        'a': ['target', 'class', 'href']
+    }
+};
+const URL_REGEX = /https?:\/\/(www\.)?[-a-zA-Z0-9@:%._+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_+.~#?&//=]*)/;
```

Comment:
Put a `g` on the end of this to make it greedy, then it will match all URLs without needing to loop.

ie.
``` JavaScript
const URL_REGEX = /https?:\/\/(www\.)?[-a-zA-Z0-9@:%._+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_+.~#?&//=]*)/g;
```